ow_testbed/launch/spawn.launch

```diff
@@ -25,6 +25,7 @@
   <!-- Convert joint states from Gazebo to tf-tree for rviz -->
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" >
     <param name="publish_frequency" value="30" />
+    <remap from="/joint_states" to="/faults2/joint_states" />

```

Comment:
This file is not used by the simulation I would suggest you undo this change to stay on what matters.